support/README.md

```diff
@@ -69,5 +69,5 @@ To run clang-tidy under Docker, run the following (this creates a full
 compilation db and takes a long time):
 
 ```console
-./ci/run_envoy_docker.sh ci/do_ci.sh bazel.clang_tidy
+./ci/run_envoy_docker.sh ci/do_ci.sh clang_tidy
```

Comment:
This appears to be an unrelated change. Idk what the maintainers' preferences are--I'm just pointing it out in case it's unintended.

Reply:
+1 can you add to a seperate PR and have phlax@ (who knows about tooling) review it

Reply:
https://github.com/envoyproxy/envoy/pull/29990